src/components/hv-section-list/index.tsx

```diff
@@ -144,19 +142,23 @@ export default class HvSectionList extends PureComponent<
         'animated',
       ) === 'true';
 
-    const viewOffset: ?number =
-      parseInt(
-        behaviorElement?.getAttributeNS(Namespaces.HYPERVIEW_SCROLL, 'offset'),
-        10,
-      ) || undefined;
-
-    const viewPosition: ?number =
-      parseFloat(
-        behaviorElement?.getAttributeNS(
-          Namespaces.HYPERVIEW_SCROLL,
-          'position',
-        ),
-      ) || undefined;
```

Comment:
while this might be a code improvement, this adds surface to the diff, let's see if that can be reverted

Reply:
57970c0